src/test/java/core/basesyntax/operation/BalaceOperationHandlerTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalaceOperationHandlerTest {
+    private static final FruitTransaction fruitTransaction
+            = new FruitTransaction("b", new Fruit("banana"), 10);
+    private static final Fruit fruit = new Fruit("banana");
```

Comment:
let's make them either constants (fix name) or local variables
same in other classes